datadog_checks_dev/datadog_checks/dev/tooling/templates/jmx/{check_name}/datadog_checks/{check_name}/data/conf.yaml.example

```diff
@@ -1,4 +1,14 @@
 init_config:
 
```

Comment:
By the way, if I understand correctly the following is required for every jmx integraiton. Can we add it in the template?

```  ## @param is_jmx - boolean - required
  ## Whether or not this file is a configuration for a JMX integration
  #
  is_jmx: true

  ## @param collect_default_metrics - boolean - required
  ## Whether or not the check should collect all default metrics for this integration.
  #
  collect_default_metrics: true```

Reply:
Added